papers/extract.py

```diff
@@ -120,7 +120,8 @@ def isvaliddoi(doi):
 
 
 def pdfhead(pdf, maxpages=10, minwords=200, image=False):
-    """ read pdf header
+    """
```

Comment:
I don't mind, but I also don't see anything [here](https://peps.python.org/pep-0257) that favours one over the other.
If anything, one liner `""" read pdf header"""`

Reply:
Yeah, you're right -- I guess I don't actually know where the fetish for 
"""
thing
"""
 came from...